### PR TITLE
Examples: Disable alpha buffer when not required

### PR DESCRIPTION
**Pull request by WestLangley**
I think this is an appropriate change in these examples...

### CODE REVIEW

File: examples/webgl_lensflares.html [hunk 1]
```diff
@@ -112,7 +112,7 @@
 
 				// renderer
 
-				renderer = new THREE.WebGLRenderer( { antialias: true, alpha: true } );
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
```
Comment (Mugen87) [inline examples/webgl_lensflares.html:115]: I remember this was set on purpose my @mrdoob for performance reasons. I'll try to find the related issue/PR.

Comment (Mugen87) [reply examples/webgl_lensflares.html:115]: Unfortunately, I find only the commit but not the discussion: 1937e58e8eedff74f2a8c2d3274a56c098cddc38

Comment (Mugen87) [reply examples/webgl_lensflares.html:115]: Here is the discussion 😇 https://github.com/mrdoob/three.js/issues/13170#issuecomment-365142026

Comment (mrdoob) [reply examples/webgl_lensflares.html:115]: Hmm, we can always revert if needed.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**